Deployment/Jenkins/master/parent-instance.groovy

```diff
@@ -0,0 +1,303 @@
+                        "ParameterValue": "${env.NoUpdates}"
+                      },
+                      {
+                        "ParameterKey": "PipIndexFips",
+                        "ParameterValue": "${env.PipIndexFips}"
+                      },
+                      {
+                        "ParameterKey": "PipRpm",
+                        "ParameterValue": "${env.PipRpm}"
+                      },
+                      {
+                        "ParameterKey": "ProvisionUser",
+                        "ParameterValue": "${env.ProvisionUser}"
+                      },
+                      {
+                        "ParameterKey": "PyStache",
+                        "ParameterValue": "${env.PyStache}"
+                      },
+                      {
+                        "ParameterKey": "ServerHostname",
+                        "ParameterValue": "${env.ServerHostname}"
+                      },
+                      {
+                        "ParameterKey": "ServiceTld",
+                        "ParameterValue": "${env.ServiceTld}"
+                      },
+                      {
+                        "ParameterKey": "AdminPubkeyURL",
+                        "ParameterValue": "${env.AdminPubkeyURL}"
+                      },
+                      {
+                        "ParameterKey": "BackupFolder",
+                        "ParameterValue": "${env.BackupFolder}"
+                      },
+                      {
+                        "ParameterKey": "BucketTemplate",
+                        "ParameterValue": "${env.BucketTemplate}"
+                      },
+                      {
+                        "ParameterKey": "Ec2Template",
+                        "ParameterValue": "${env.Ec2Template}"
+                      },
+                      {
+                        "ParameterKey": "ElbTemplate",
+                        "ParameterValue": "${env.ElbTemplate}"
+                      },
+                      {
+                        "ParameterKey": "IamRoleTemplate",
+                        "ParameterValue": "${env.IamRoleTemplate}"
+                      },
+                      {
+                        "ParameterKey": "SecurityGroupTemplate",
+                        "ParameterValue": "${env.SecurityGroupTemplate}"
+                      },
+                      {
+                        "ParameterKey": "TargetVPC",
+                        "ParameterValue": "${env.TargetVPC}"
+                      },
+                      {
+                      "ParameterKey": "BackupCronURL",
+                      "ParameterValue": "${env.BackupCronURL}"
+                      },
+                      {
+                      "ParameterKey": "SubnetId",
+                      "ParameterValue": "${env.SubnetId}"
+                      }
+                    ]
+                   /
+                }
+            }
+        stage ('Prepare AWS Environment') {
```

Comment:
This stage now, essentially, does nothing. Might want to wholly excise the stage or at least remove any functional elements that could cause accidental damage (i.e., the `delete-stack` operation). Either that, or make the `delete-stack`'s target match the `create-stack`'s (lines 271, 272 and 274).

Reply:
Done